global_costmap:
    footprint: [[-0.1, -0.1], [-0.1, 0.1], [0.1, 0.1], [0.1, -0.1]]
    footprint_padding: 0.01
    transform_tolerance: 0.5
    update_frequency: 20.0
    publish_frequency: 20.0

    global_frame: map
    robot_base_frame: base_footprint
    resolution: 0.05

    rolling_window: false
    track_unknown_space: true

    plugins: 
        - {name: static,           type: "costmap_2d::StaticLayer"}            
        #- {name: sensor,           type: "costmap_2d::ObstacleLayer"}
        - {name: swarm_layer,      type: "ddsbot_swarm_layer::SwarmLayer"}
        - {name: inflation,        type: "costmap_2d::InflationLayer"}

    static:        
        map_topic: /map 
        subscribe_to_updates: true

    #sensor:
        #observation_sources: base_scan
        #base_scan: {sensor_frame: laser, data_type: LaserScan, topic: scan, marking: true, clearing: true}

    swarm_layer:
       enabled: true

    inflation:
        inflation_radius: 0.55
        cost_scaling_factor: 7.0
